Interface/Employees.java

```diff
@@ -0,0 +1,9 @@
+package Interface;
+
+public interface Employees extends EmployeesNumber{
```

Comment:
Looks like there is no inheritance connection between Employees and EmployeesNumber